### ISSUE TITLE
Reopening Atom forgets read-only status

### ISSUE THREAD

**Issue by seth10** [deprint/toggle-read-only#3]
Good morning!

I have a suggestion. When I open Atom at the beginning of the day, it opens the last set of files I was looking at. However, they are no longer marked read-only. This could lead to some significant accidents. I would love to see this package returns files' read-only status to what it was when I closed Atom. Unfortunately, I don't know if this is trivial or near impossible.

Thank you for your time!

**Comment by fstiewitz** [comment]
Thanks for the suggestion!
I added it to `v0.1.6`. Let me know if there are any problems.

**Comment by seth10** [comment]
It works perfectly, thank you so much!

**Comment by seth10** [comment]
I've actually ran in to some issues of files opening read-only when they should. As in they were closed and I double-clicked the file name to open it in a tab, and it was set to read-only. I don't think it was set to read-only when I last closed the tab for that file. I haven't been able to recreate the situation reliably, if I do I'll post back here. I could be derping, but just wanted to make you aware in case there is an issue.
